great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -1038,3 +1076,46 @@ def add_query_asset(
             batch_metadata=batch_metadata or {},
         )
         return self._add_asset(asset)
+
+
+def _verify_table_name_exists_and_get_normalized_typed_name_map(
+    name: str,
+    typed_names: List[str | sqlalchemy.quoted_name],
+    error_message_template: str = 'Error: The table "{table_name:s}" does not exist.',
+    verify_only: bool = False,
```

Comment:
are any callers using these parameters? if not, let's remove them - no need to overcomplicate a util if we can always modify it later to allow for more flexibility